libs/gisday/platform/ngx/core/src/lib/pages/admin/pages/university/admin-add-university/admin-add-university.component.scss

```diff
@@ -0,0 +1,13 @@
+@import 'libs/sass/mixins';
+
+label {
```

Comment:
Rule doesn't do anything. I think you're trying to fix the vertical positioning because the labels are clipping into the textboxes but this rules does not penetrate into the textbox component and this is not the place to alter the style for all labels.

There are plenty of styles that are interfereing with the component styles in the global app styles.scss

Reply:
I was only trying to modify the style of the labels within that component, is that the way to do so?